kolibri/core/assets/src/utils/navItems.js

```diff
@@ -0,0 +1,11 @@
+export let menuItems = [];
+
+export function setMenuItems(items) {
+  menuItems = items;
+}
+
+export let userMenuItems = [];
+
+export function setUserMenuItems(items) {
+  userMenuItems = items;
+}
```

Comment:
This might be fine, yeah - I will check, I did it like this, as I recall having weird issues with assigning to module exports from outside the module.